xi-win-shell/src/util.rs

```diff
@@ -251,3 +260,21 @@ macro_rules! accel {
         ]
     }
 }
+
+pub fn attach_console() {
+    unsafe {
+        if AttachConsole(ATTACH_PARENT_PROCESS) == 0 || AllocConsole() == 0 {
+            let chnd = CreateFileA(
```

Comment:
Is there any chance this function can fail? Seems like maybe we should check an error condition.